fedimint-server-core/src/init.rs

```diff
@@ -44,6 +45,7 @@ pub trait IServerModuleInit: IDynCommonModuleInit {
         db: Database,
         task_group: &TaskGroup,
         our_peer_id: PeerId,
+        module_api: DynModuleApi,
```

Comment:
It was supposed to be simpler, now we need to inject DynModuleApi to modules. :P

Reply:
Yeah, I wish we had implemented the upgrade logic on the core level :cry: